data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2467,6 +2467,7 @@ spec:
                     - AzureChinaCloud
                     - AzureGermanCloud
                     - AzureStackCloud
+                    - USSec
```

Comment:
That's not how you update this file. It's auto-generated based on the annotations of the types used. The cloud types are defined here: https://github.com/openshift/installer/blob/master/pkg/types/azure/platform.go#L124-L143.